Microsoft.Toolkit.Uwp.UI/Media/RadialGradientBrush.cs

```diff
@@ -103,5 +110,32 @@ protected override bool OnDraw(CanvasDevice device, CanvasDrawingSession session
 
             return false;
         }
+
+        private void RegisterAppResumeHandler()
+        {
+            if (_resumingHandler == null)
+            {
+                _resumingHandler = new System.EventHandler<object>(App_Resuming);
```

Comment:
Not sure if _resumingHandler is necessary, why not register the event handler directly

Reply:
I added the _resumingHandler to reference the handler and make sure that it's not being bound more than once.

Reply:
Doesn't line 118 already make sure you are not registering the handler multiple times?

Reply:
That's correct. I just refactored the handler.